packages/greencheck/src/Sitecheck.php

```diff
@@ -130,6 +131,7 @@ public function __construct(
         $this->_calledfrom = $calledfrom;
 
         $this->validator = new Validator();
+        $this->dnsFetcher = new DnsFetcher();
```

Comment:
@mrchrisadams if you pass this in from the constructor itself, you can also pass a mocked one where you define what getIpAddressesForUrl($url) should return